paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt

```diff
@@ -190,7 +190,7 @@ internal class PaymentOptionsViewModelTest {
         )
 
         viewModel.currentScreen.test {
-            assertThat(awaitItem()).isEqualTo(PaymentSheetScreen.AddAnotherPaymentMethod)
+            assertThat(awaitItem() is PaymentSheetScreen.AddAnotherPaymentMethod).isTrue()
```

Comment:
Could also use `assertThat(awaitItem()).isInstanceOf(` here, which will give you a nicer error message when it fails.

Reply:
thanks! Made that change